backend/routing/users.js

```diff
@@ -45,8 +45,20 @@ const usersController = require('../controllers/users.js')(DAL);
 
   server.route({
     method: 'POST',
+    path: '/api/invite-user',
+    config: {
+      handler: function (request, reply) {
+        reply (usersController.inviteUser(request.payload, Boom.badImplementation('Server error')));
```

Comment:
please don't send error handling as a parameter. Use code similar to '/api/user/{id}' route handler